src/Selenium2Driver.php

```diff
@@ -758,170 +695,135 @@ public function selectOption($xpath, $value, $multiple = false)
      */
     public function isSelected($xpath)
     {
-        return $this->findElement($xpath)->selected();
+        $element = $this->findElement($xpath);
+        return $element->isSelected();
     }
 
     /**
      * {@inheritdoc}
      */
     public function click($xpath)
     {
-        $this->clickOnElement($this->findElement($xpath));
+        $element = $this->findElement($xpath);
+        $this->clickOnElement($element);
     }
 
-    private function clickOnElement(Element $element)
+    private function clickOnElement(WebDriverElement $element)
     {
-        try {
-            // Move the mouse to the element as Selenium does not allow clicking on an element which is outside the viewport
-            $this->wdSession->moveto(array('element' => $element->getID()));
-        } catch (UnknownCommand $e) {
-            // If the Webdriver implementation does not support moveto (which is not part of the W3C WebDriver spec), proceed to the click
-        }
-
-        $element->click();
+        // Move the mouse to the element as Selenium does not allow clicking on an element which is outside the viewport
```

Comment:
this comment is not relevant anymore, as you removed the corresponding code

Reply:
removed, but jfyi this logic is just implemented in `\Facebook\WebDriver\Remote\RemoteMouse`